src/main/java/org/javarosa/xform/parse/FormInstanceParser.java

```diff
@@ -89,11 +89,15 @@ FormInstance parseInstance(Element e, boolean isMainInstance, String name, Map<S
 
             List<TreeElement> meta = root.getChildrenWithName("meta");
             if (!meta.isEmpty()) {
-                List<TreeElement> entities = meta.get(0).getChildrenWithName("entities:entity");
+                List<TreeElement> entity = meta.get(0).getChildrenWithName("entities:entity");
```

Comment:
Will need to add support for the `http://www.opendatakit.org/xforms/entities` namespace and a way to get children by namespace and name.